eng/scripts/live-test-resource-cleanup.ps1

```diff
@@ -67,7 +67,8 @@ Write-Host "Count $($hasDeleteAfter.Count)"
 $toDelete = $hasDeleteAfter.Where({ $deleteDate = ($_.Tags.DeleteAfter -as [DateTime]); (!$deleteDate -or $now -gt $deleteDate) })
 Write-Host "Groups to delete: $($toDelete.Count)"
 
-$purgeableResources = @()
+# Get purgeable resources arleady in a deleted state.
```

Comment:
```suggestion
# Get purgeable resources already in a deleted state.
```
Any reason to do this before cleaning up the resource groups? I moved it down because I thought it would allow the resource group clean-up to finish before it even tries to retrieve these.

Reply:
It'd be redundant. `Get-PurgeableGroupResources` will get them, so they'd be double-counted. `Get-PurgeableResources` only gets *deleted* resources. The expectation is that, normally, KVs and MHSMs in a group that isn't deleted also aren't yet deleted.